app/assets/stylesheets/application.scss

```diff
@@ -17,6 +17,11 @@
 
  */
 
+  #move-video-big {
```

Comment:
Avoid using id selectors<br>Line should be indented 0 spaces, but was indented 2 spaces